### ISSUE TITLE
fix t02022 failures seen on Darwin

### ISSUE THREAD

**Issue by nigoroll** [varnishcache/varnish-cache#3733]
reminder because ENOTIME rn:
http://varnish-cache.org/vtest/attachment_64f7181b1d337309e2bcae8a10be5f626f009e36_Darwin_arm64_20.5.0_1.05__phk_t02022.txt coincided with 64f7181b1d337309e2bcae8a10be5f626f009e36

Is it related to the change in `h2_win_limit()`?

**Comment by Dridi** [comment]
```
---- c3.1  Frame #1 for rxresp was of type RST_STREAM (3) instead of HEADERS (1)
```
It needs a stabilization similar to c8d0781df30107481b3b199995973eee547dd034.

**Comment by nigoroll** [comment]
Sounds good, thank you.
But I am still puzzled by the coincidence

**Comment by Dridi** [comment]
I think the coincidence is the new darwin entry in our vtest collection.

**Comment by bsdphk** [comment]
fixed with 271e2b8307ff56942c
